lib/utils/createElement.js

```diff
@@ -9,8 +10,8 @@ export function toggleButton(
 	onClick?: (enabling: boolean) => void = () => {},
 	fieldID: string,
 	enabled?: boolean = false,
-	onText?: string | HTMLElement = 'on',
-	offText?: string | HTMLElement = 'off',
+	onText?: string | HTMLElement = i18n('toggleOn'),
```

Comment:
The question is whether we even need onText and offText, they are not overwritten anywhere in the code, it seems no feature in the nearest future could use it. Personally, I'd just remove them and call i18n on toggleOn directly in rendering function `$(<span>....)`.